test/mocha_setup.js

```diff
@@ -0,0 +1,31 @@
+var sinon = require('sinon');
+var autoRelease = require('auto-release-sinon');
+
+require('babel/register')(require('../src/optimize/babelOptions').node);
+
+var queuedAfterEachArgs = [];
+Object.defineProperty(global, 'afterEach', {
```

Comment:
Comments explaining the purpose of this would be helpful. I don't know what auto-release-sinon is or why it needs to be set up in this way.